modules/highlight/highlight.go

```diff
@@ -119,9 +120,13 @@ func File(numLines int, fileName string, code []byte) map[int]string {
 
 	lexer := lexers.Match(fileName)
 	if lexer == nil {
-		lexer = lexers.Analyse(string(code))
+		mimeType := mimetype.Detect(code)
```

Comment:
imho we should reuse already existing logic we have to detect languages:
```suggestion
		language := analyze.GetCodeLanguage(fileName, code)
		mimeType := enry.GetMIMEType(fileName, language)
```

Reply:
maybe even we could remove `lexers.Match(fileName)`

Reply:
Didn't know about that I'll look into it thanks!

Reply:
Thanks @lafriks using existing module worked great!